Marlin/src/gcode/feature/mixing/M163-M165.cpp

```diff
@@ -105,6 +105,7 @@ void GcodeSuite::M164() {
     if (mix_bits) {
       MIXER_STEPPER_LOOP(i)
         if (!TEST(mix_bits, i)) mixer.set_collector(i, 0.0f);
+      mixer.T(MIXER_DIRECT_SET_TOOL);
       mixer.normalize();
```

Comment:
I don't know the use cases for `M165`, but I do feel that this change in its fundamental behavior is liable to break expectations. I should be able to use `M165` and then go to the Mixing menu in the LCD and see that my current tool index has not changed, and that my mix is still set to the new values.

Reply:
I will change this back